### PR TITLE
Explain choice of 50% for residue removal

### PR DESCRIPTION
**Pull request by samsrabin**
### Description of changes

Simplifies and corrects various things about the biofuel harvest and residue removal documentation. Probably the most important is to explain the reasoning behind the default 50% residue removal.

### Specific notes

**Contributors other than yourself, if any:** None

**CTSM Issues Fixed (include github issue #):** None

**Are answers expected to change (and if so in what way)?** No

**Any User Interface Changes (namelist or namelist defaults changes)?** No

**Testing performed, if any:** Builds fine on my machine, in both VSCode and with the command-line Docker-based method.

### CODE REVIEW

File: doc/source/tech_note/Crop_Irrigation/CLM50_Tech_Note_Crop_Irrigation.rst [hunk 1]
```diff
@@ -553,64 +560,6 @@ with corresponding nitrogen fluxes:
 
 where CF is the carbon flux, CS is stored carbon, NF is the nitrogen flux, NS is stored nitrogen, and :math:`biofuel\_harvfrac` is the harvested fraction of leaf/livestem for biofuel feedstocks.
 
-.. _Table Plant functional type (PFT) parameters for harvested fraction of leaf/livestem for bioenergy production:
```
Comment (ekluzek) [inline doc/source/tech_note/Crop_Irrigation/CLM50_Tech_Note_Crop_Irrigation.rst:556]: @samsrabin I'm just not sure why this table is being removed?

File: doc/source/tech_note/Crop_Irrigation/CLM50_Tech_Note_Crop_Irrigation.rst [hunk 2]
```diff
@@ -553,64 +560,6 @@ with corresponding nitrogen fluxes:
...
- | Tropical Corn                    |             0.00           |
- +----------------------------------+----------------------------+
- | Tropical Soybean                 |             0.00           |
- +----------------------------------+----------------------------+
- | Miscanthus                       |             0.70           |
- +----------------------------------+----------------------------+
- | Switchgrass                      |             0.70           |
- +----------------------------------+----------------------------+
-
-Whereas food C and N was formerly transferred to the litter pool, CLM5 routes food C and N to a grain product pool where the C and N decay to the atmosphere over one year, similar in structure to the wood product pools. Biofuel and removed-residue C and N is also routed to the grain product pool and decays to the atmosphere over one year. Additionally, CLM5 accounts for the C and N required for crop seeding by removing the seed C and N from the grain product pool during harvest. The crop seed pool is then used to seed crops in the subsequent year.
```
Comment (ekluzek) [inline doc/source/tech_note/Crop_Irrigation/CLM50_Tech_Note_Crop_Irrigation.rst:612]: Shouldn't this stay as well?

Comment (ekluzek) [reply doc/source/tech_note/Crop_Irrigation/CLM50_Tech_Note_Crop_Irrigation.rst:612]: Above paragraph that is...

**Review by ekluzek — COMMENTED** [review]
I'm just wondering about why the table and paragraph that follows it was removed?

File: doc/source/tech_note/Crop_Irrigation/CLM50_Tech_Note_Crop_Irrigation.rst (hunk 2 above)
Comment (samsrabin) [reply doc/source/tech_note/Crop_Irrigation/CLM50_Tech_Note_Crop_Irrigation.rst:612]: A lot of info in that paragraph was redundant with text already included above. I added text above to cover the rest.

File: doc/source/tech_note/Crop_Irrigation/CLM50_Tech_Note_Crop_Irrigation.rst (hunk 1 above)
Comment (samsrabin) [reply doc/source/tech_note/Crop_Irrigation/CLM50_Tech_Note_Crop_Irrigation.rst:556]: It's an inefficient use of space. The information is now more succinctly conveyed at line 517:
> For the biofuel crops Miscanthus and switchgrass, 70% of live leaf and stem biomass at harvest is transferred to the crop product pool as described for "food" harvest above. This value can be changed for these crops—or set to something other than the default zero for any other crop—with the parameter `biofuel_harvfrac` (0-1).

Comment (ekluzek) [reply doc/source/tech_note/Crop_Irrigation/CLM50_Tech_Note_Crop_Irrigation.rst:556]: Oh, you are right there is a table where everything other than the last two are zeros! No need for that.

**Review by ekluzek — APPROVED** [review]

**Pull request merged**